### ISSUE TITLE
branch coverage support

### ISSUE THREAD

**Issue by skcc321** [dashingrocket/simplecov-cobertura#17]
I see no support for branch coverage in the readme. But I would like to start the process of implementation of that one somehow.
according to cobertura documentation - branch coverage is supported.
https://www.baeldung.com/cobertura#2-configuring-code-coverage-calculation-and-checks
simplecov supports branch coverage as well
https://github.com/simplecov-ruby/simplecov#branch-coverage-ruby--25

let's implement that one finally :)

**Comment by jessebs** [comment]
Great find.  Is that something you'd want to take up?

**Comment by skcc321** [comment]
probably. but not sure how soon...

**Comment by ms-ati** [comment]
Can anyone provide guidance on how to implement branch coverage, please?

**Comment by skcc321** [comment]
I think this might be an example https://github.com/leobalter/testing-examples/blob/master/solutions/3/report/cobertura-coverage.xml how it should look like.

**Comment by klausbadelt** [comment]
Istanbul reports branches like this:
```xml
<line number="38" hits="59" branch="true" condition-coverage="100% (2/2)"/>
```

Has anyone picked this up yet? I might have a go otherwise.

**Comment by jessebs** [comment]
As far as I am aware, no one has picked this up

**Comment by skcc321** [comment]
nobody....

**Comment by skcc321** [comment]
something like this https://github.com/dashingrocket/simplecov-cobertura/pull/20

**Comment by jessebs** [comment]
Thank you for the work on this and I'm sorry for the delay.  This is merged in and will be released as part of 2.0.0
